ibm/service/power/resource_ibm_pi_cloud_connection_network_attach.go

```diff
@@ -36,19 +35,19 @@ func ResourceIBMPICloudConnectionNetworkAttach() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			// Required Attributes
-			helpers.PICloudInstanceId: {
+			Arg_CloudInstanceID: {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
 				Description: "PI cloud instance ID",
```

Comment:
Sort these fields (type, requried, forcenew description) alphabetically, and match their description to what's in the documentation.